### PR TITLE
Update ruby, rails, spring, rollbar

### PR DESCRIPTION
**Pull request by ArturMinnullin**
@timurvafin Please, change ruby version in semaphore.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Upgrade Rails to 4.2.6, Ruby to 2.3.0, Rollbar to 2.8.3, Spring to 1.6.4
```
Comment (artembiserov) [inline CHANGELOG.md]: please, remove lines below about upgrading rails and ruby
is it necessary to add info about upgrading gems like rollbar and spring to changelog?

Comment (ArturMinnullin) [inline CHANGELOG.md]: done

**Comment by ArturMinnullin** [comment]
@timurvafin please, update ruby version

**Comment by timurvafin** [comment]
@ArturMinnullin https://semaphoreci.com/docs/languages/ruby/switching-ruby-version-on-specific-branch.html

**Pull request merged**